src/memory/FontEnlarger.java

```diff
@@ -0,0 +1,9 @@
+package memory;
+
+import java.awt.*;
```

Comment:
Avoid to use *, it is not a good practice

Reply:
done